tools/export/cmsis/cpdsc.tmpl

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
You didn't mention that CPDSC support was coming in with this PR, did you mean to include it?